tests/_runtime/test_runtime.py

```diff
@@ -881,11 +881,11 @@ async def test_run_scratch(mocked_kernel: MockedKernel) -> None:
         assert all(m[1]["cell_id"] == SCRATCH_CELL_ID for m in messages)
         assert m1[1]["status"] == "queued"
         assert m2[1]["status"] == "running"
-        assert m3[1]["status"] == "idle"
-        assert m4[1]["status"] is None
+        assert m3[1]["status"] is None
         assert (
-            m4[1]["output"]["data"] == "<pre style='font-size: 12px'>1</pre>"
+            m3[1]["output"]["data"] == "<pre style='font-size: 12px'>1</pre>"
         )
+        assert m4[1]["status"] == "idle"
```

Comment:
oh this fixes also what i thought was a bug which is "why are there more idle messages, if its already idle". its clearer/cleaner that "running" is repeated